spec/reset_password_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+describe 'password reset', type: :feature, js: true do
+  it 'resets password at LOA1' do
+    email_address = create_new_account
+
+    visit idp_logout_url
+
+    visit idp_reset_password_url
+
+    expect(page).to have_content('Forgot')
```

Comment:
I am not sure we *need* this expectation given that the following line fills in a form that would only be present on the password reset page